src/App.jsx

```diff
@@ -1,3 +1,46 @@
-import React from "react";
+import React, {useState} from "react";
+import { Form } from "./Form";
 
-export const App = () => <div>Empty page</div>
\ No newline at end of file
+export const App = () => {
+
+    const [users, setUsers] = useState([])
```

Comment:
It's a very bad idea to keep data in an array. You can not predict all the cases of placement data in the indexes. Then we have problems like this:
![image](https://github.com/KAR-HUD-SOFTWARE-DEVELOPMENT/user-form/assets/49559731/dcf82725-c900-432c-8e54-594fcd1508ba)
![image](https://github.com/KAR-HUD-SOFTWARE-DEVELOPMENT/user-form/assets/49559731/1b726fb6-1049-41ca-a87b-64c45209e666)

Instead of array use object: with keys { name: 'Karolina", surname: 'Hudziec'....}